src/utils/SdkCoin.sol

```diff
@@ -109,7 +109,7 @@ library SdkCoin {
      * @param amount The amount to be converted
      * @return convertedAmount The amount converted to uint256 supported by ERC20 tokens
      */
-    function _SdkCoinToERC20_ConvertAmount(address tokenAddress, uint64 amount) internal view returns (uint256) {
+    function convertAmountSdkCoinToERC20(address tokenAddress, uint64 amount) internal view returns (uint256) {
```

Comment:
Do you think we should name these `convertAmountFromERC20` and `convertAmountToERC20` so that it is not so long?

```suggestion
    function convertAmountToERC20(address tokenAddress, uint64 amount) internal view returns (uint256) {
```